tekton/get-version.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-versions
+spec:
+  resources:
+    inputs:
+    - name: source
```

Comment:
For consistency with the Pipeline, can this be `source-repo`? That or change this PipelineResource on the Pipeline to `source`?

Reply:
Yeah :wink: